regression/cbmc/memset_null_02/main.c

```diff
@@ -0,0 +1,8 @@
+#include <string.h>
```

Comment:
I removed `memset_null_01` since it's subsumed (during symex) by `memset_null_02`, but the other test was necessary and now I have split the tests into two different commits, as per your suggestion.